bootstrap.sh

```diff
@@ -110,7 +110,7 @@ done
 
 # Symlink /opt/genestack/kustomize.sh to
 # /etc/genestack/kustomize/kustomize.sh
-ln -s $base_source_dir/base-kustomize/kustomize.sh $base_target_dir/kustomize/kustomize.sh
+ln -s $base_source_dir/base-kustomize/kustomize.sh $base_target_dir/kustomize.sh
```

Comment:
```suggestion
ln -s $base_source_dir/kustomize.sh $base_target_dir/kustomize.sh
```

reopening this one, we need to make one more change to the symlink.

Reply:
ok

Reply:
Done